### ISSUE TITLE
Push to Conda as well as PyPi

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/.github#15]
## Detailed Description
Conda is used by a lot of scientific software and is the recommended way of installing a lot of software that we work with/use. It could be nice to also be able to install the nowcasting packages through conda-forge 

## Context
Conda helps make sure python environments all work together, and is a common way of installing packages, etc for ML and scientific software, so making our packages available through there as well could be nice.

## Possible Implementation
A GH Action like for PyPi publishing. Docs for contributing to conda-forge are here: https://conda-forge.org/docs/maintainer/adding_pkgs.html

**Comment by JackKelly** [comment]
Great idea!

Also, we should check how best to install each OCF Python package using conda:  at the moment we ask users to `conda env create -f environment.yml` and then install the OCF package into the conda env by running `pip install -e .` which works but forces pip to install a bunch of redundant packages (because they've already been installed by conda).  I'm assuming there must be a way to install each OCF Python package using conda?

**Comment by jacobbieker** [comment]
Yeah, that's what this should allow. So ideally, we could then just do something like `conda create -n ocf python=3.9 pytorch satflow -c pytorch -c conda-forge -c nvidia` and it would get everything correct without us needing to use pip at all. Currently, we can't use conda because its not in conda-forge or any conda channel right now, so we'd have to do a little bit of work to get it going, but I think it would be worth it as conda has a lot of nice features. Just a bit more work to setup than PyPi, which is why I've not tried doing it yet

**Comment by jacobbieker** [comment]
There is a github action for it: https://github.com/marketplace/actions/publish-conda-package-to-anaconda-org
